nilearn/externals/skimage/peak.py

```diff
@@ -0,0 +1,161 @@
+import numpy as np
+import scipy.ndimage as ndi
+from _rank_order import rank_order
```

Comment:
I also got reviews from @GaelVaroquaux on my overall PR. I am also fixing those. Then, may be you can take a look.